t/151-ssl-ctx.t

```diff
@@ -0,0 +1,73 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use Test::Nginx::Socket::Lua;
+use Digest::MD5 qw(md5_hex);
+
+repeat_each(3);
+
+plan tests => repeat_each() * (blocks());
+
+$ENV{TEST_NGINX_HTML_DIR} ||= html_dir();
+
+log_level 'debug';
+
+no_long_string();
+
+add_block_preprocessor(sub {
+    my $block = shift;
+
+    if (!defined $block->user_files) {
+        $block->set_value("user_files", <<'_EOC_');
+>>> defines.lua
+local ffi = require "ffi"
+
+ffi.cdef[[
+    void *ngx_http_lua_ffi_ssl_ctx_init(unsigned int protocols, char **err);
+
+    void ngx_http_lua_ffi_ssl_ctx_free(void *cdata);
+
+    int ngx_http_lua_ffi_ssl_ctx_set_priv_key(void *cdata_ctx,
+        void *cdata_key, char **err);
+
+    int ngx_http_lua_ffi_ssl_ctx_set_cert(void *cdata_ctx,
+        void *cdata_cert, char **err);
+]]
```

Comment:
Well. The only thing I'm confused  is what's the 'simply test' ?  if we want to simply test the `ngx_http_lua_ffi_ssl_ctx_set_priv_key` and `ngx_http_lua_ffi_ssl_ctx_set_cert` , should we set the priv_key and cert then send it to server ? Or just set the priv_key and cert then testing the return value.